channeldb/kvdb/config.go

```diff
@@ -1,18 +1,31 @@
 package kvdb
 
-// BoltBackendName is the name of the backend that should be passed into
-// kvdb.Create to initialize a new instance of kvdb.Backend backed by a live
-// instance of bbolt.
-const BoltBackendName = "bdb"
+import "time"
 
-// EtcdBackendName is the name of the backend that should be passed into
-// kvdb.Create to initialize a new instance of kvdb.Backend backed by a live
-// instance of etcd.
-const EtcdBackendName = "etcd"
+const (
+	// BoltBackendName is the name of the backend that should be passed into
+	// kvdb.Create to initialize a new instance of kvdb.Backend backed by a
+	// live instance of bbolt.
+	BoltBackendName = "bdb"
+
+	// EtcdBackendName is the name of the backend that should be passed into
+	// kvdb.Create to initialize a new instance of kvdb.Backend backed by a
+	// live instance of etcd.
+	EtcdBackendName = "etcd"
+
+	// DefaultBoltAutoCompactMinAge is the default minimum time that must
+	// have passed since a bolt database file was last compacted for the
+	// compaction to be considered again.
+	DefaultBoltAutoCompactMinAge = time.Hour * 6
+)
```

Comment:
is there a reason 6 hours was chosen here? thinking that as a default value on mobile, maybe it's not ideal to do this every time the app is opened. for larger nodes which typically have longer uptimes and less frequent restarts, the difference should be neglible

Reply:
Yes, 6 hours was probably a bit short. Changed it to a week.